instrumentation/cassandra/cassandra-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v4_0/CassandraAttributesExtractor.java

```diff
@@ -36,6 +38,12 @@ public void onEnd(
 
     Node coordinator = executionInfo.getCoordinator();
     if (coordinator != null) {
+      SocketAddress address = coordinator.getEndPoint().resolve();
+      if (address instanceof InetSocketAddress) {
+        attributes.put(
+            SemanticAttributes.SERVER_ADDRESS, ((InetSocketAddress) address).getHostName());
```

Comment:
`InetSocketAddress` `getHostName` may cause a `DNS` lookup, we use `getHostString` instead.